.github/PULL_REQUEST_TEMPLATE/develop-template.md

```diff
@@ -0,0 +1,16 @@
+Originator(s):
+
+Summary (include the keyword ['closes', 'fixes', 'resolves'] and issue number):
+
+Describe any changes made to the namelist:
+
+List all files eliminated and why:
+
+List all files added and what they do:
+
+List all existing files that have been modified, and describe the changes: 
+(Helpful git command: `git diff --name-status development...<your_branch_name>`)
+
```

Comment:
Add additional bullet like `List how this code was validated with the new/modified features`  - i.e. How does the developer know that it is doing the proper thing?